### ISSUE TITLE
Int conversion issues when showing plot grid

### ISSUE THREAD

**Issue by ntjess** [pyqtgraph/pyqtgraph#1484]
In '0.11.1.dev0' (and 0.11.1), the following code will issue an error:

```python
import pyqtgraph as pg
from warnings import simplefilter
simplefilter('error', DeprecationWarning)

pw = pg.PlotWindow()
pi = pw.plotItem
pi.showGrid(True, True, alpha=0.5)
#   File "C:\Users\njessurun\AppData\Local\Continuum\anaconda3\envs\py38\lib\site-# packages\pyqtgraph\graphicsItems\PlotItem\PlotItem.py", line 382, in showGrid
#     self.ctrl.gridAlphaSlider.setValue(v)
# TypeError: setValue(self, int): argument 1 has unexpected type 'numpy.float64'
```

Qt library: 'PyQt5 5.14.2 Qt 5.14.2'

**Comment by j9ac9k** [comment]
Fixed with b058d03

**Comment by ntjess** [comment]
Still seeing this error on 0.12
